Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,36 @@
+//E.g. sum([10, 20, 30]), target output: 60
+//E.g. sum(['hey', 10, 'hi', 60, 10]), target output: 80 (ignore any non-numerical elements)
+// Other methods we can use to sum are: for loop, reduce, forEach() and for...of loop
+
+
 function sum(elements) {
+  const numbers = elements.filter(ele => typeof ele === "number");
+    //console.log(numbers)
+  if (numbers.length === 0) {
+    return 0;
+  }
+
+  let sum = 0;
+  numbers.forEach((el) => (sum += el));
+  //console.log(sum);
+  return sum;
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
sum([Infinity, 1]);
sum([Infinity, Infinity]);
sum([NaN, 1]);
sum([Infinity, -Infinity]);
```